src/Parser/CommandInfo.php

```diff
@@ -380,6 +380,9 @@ public function getDescription()
      */
     public function setDescription($description)
     {
+        if (!$description) {
+            return $this;
```

Comment:
This is also a "band-aid" fix we might want to give a touch more thought before merging.